docs/api/resource.rst

```diff
@@ -180,6 +189,15 @@ If a conflict occurs, an error response is returned with status ``409``.
 A ``existing`` attribute in the response gives the offending record.
 
 
+HTTP Status Codes
+-----------------
+
+* ``200 OK``: This record already exists, here is the one stored on the database;
```

Comment:
@Natim, then we should document the way it is currently implemented, and open an issue to adress this.